lib/board.rb

```diff
@@ -0,0 +1,87 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    # @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    @cells = Array.new(9, " ")
+  end
+
+  def position(position_number)
```

Comment:
Is there a way we could abstract this a lay deeper?

maybe with 

def position (position_number)
  @cells[position.to_i - 1]
end